meinberlin/apps/projectcontainers/dashboard.py

```diff
@@ -41,6 +42,15 @@ def get_urls(self):
         )]
 
 
+class ContainerInformationComponent(ProjectInformationComponent):
```

Comment:
it would be better to duplicate the ProjectInformationComponent

there are just 4 additional properties to set, which would make the ContainerInformationComponent self-contained (weight, label, form_title and form_template). 

currently the page title and the h1 are still saying "edit project information".

Reply:
That was intentional